### DISCUSSION TITLE
If I select a file in FL's search results and right-click, its context menu uses one of two different themes (It changes the theme randomly.)

### DISCUSSION THREAD

**Discussion by raindropsfromsky** [Flow-Launcher/Flow.Launcher#1197] [General]
**Describe the bug/issue**

**To Reproduce**
1. Right click on any file in the search results
2. Watch the context menu
3. The context menu can have one of two themes, as shown in the screenshots below.

**Screenshots**
![image](https://user-images.githubusercontent.com/9047168/167312023-ef0f5768-d73a-4a41-aeb0-582e4a22078e.png)
![image](https://user-images.githubusercontent.com/9047168/167312039-1b91bce5-2fc3-4008-b07e-608aa12a0297.png)

**Your System**
Windows 10 Home 64 bit 
Version	10.0.19044 Build 19044

**Flow Launcher Error Log**

**Comment by deefrawley** [comment]
Those are two different context menus for two different plugins (Explorer and Everything). The last item in the menu shows that. If you have two or more plugins with no keyword then you will get their results by default. If you want to separate them, then create a keyword for say the Everything plugin and then the Explorer will be default and you can specify when to use Everything.

**Comment by raindropsfromsky** [comment]
OK, but then IMHO all plugins should be using the same theme as set in the **Settings**. 
They should not use their own themes and icons.

**Reply by jjw24** [reply]
I am not sure what you mean by theme but in terms of icons yes it's a good idea but will be hard to implement as plugins are developed independently from flow. We can make it consistent across the default plugins that are shipped with flow and make a recommendation to plugin developers to follow the same style of icons but it will still be up to them to implement.

**Reply by jjw24** [reply]
Probably also another point to be aware with icons is that there may not be a suitable icon for a given result in the consistent icon style range as we don't make our icons.

**Comment by deefrawley** [comment]
Let's convert this to a discussion so the team can read your suggestion and see what we might be able to do.
